ElectronClient/app/app.js

```diff
@@ -970,6 +983,36 @@ class Application extends BaseApplication {
 					label: _('Focus'),
 					screens: ['Main'],
 					submenu: focusItems,
+				}, {
+					type: 'separator',
+					screens: ['Main'],
+				}, {
+					label: _('Actual Size'),
+					click: () => {
+						this.store().dispatch({
+							type: 'WINDOW_CONTENT_ZOOM_FACTOR_SET',
+							zoomFactor: 1,
+						});
+					},
+					accelerator: 'CommandOrControl+0',
+				}, {
+					label: _('Zoom In'),
+					click: () => {
+						this.store().dispatch({
+							type: 'WINDOW_CONTENT_ZOOM_FACTOR_SET',
+							zoomFactor: webFrame.getZoomFactor() + 0.1,
```

Comment:
To keep a [unidirectional data flow](https://redux.js.org/basics/data-flow) and to have only one source of truth, the new state should be based on the old one. So it would be better to use `zoomFactor: this.state.windowContentZoomFactor + 0.1`.